packages/manager/apps/catalog/src/components/Error/ErrorButtons.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsButton } from '@ovhcloud/ods-components/react';
+import { ODS_BUTTON_VARIANT } from '@ovhcloud/ods-components';
+
+export const ErrorButtons = () => {
```

Comment:
Where do you use this component inside catalog ?

Reply:
It got included back from rebase.
Gonna check.